### PR TITLE
update gemfile

### PR DESCRIPTION
**Pull request by mbarnett**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -57,6 +57,7 @@ gem 'sitemap', git: 'https://github.com/ualbertalib/rails-sitemap.git'
 gem 'rest-client'
 
 gem 'noid', '~> 0.8'
+gem 'nokogiri', '~> 1.7.1'
```
Comment (murny) [inline Gemfile]: Just a quick comment about this is for locking nokogiri or something for security reasons? As we ourselves are not using nokogiri in this project that I know of?

Comment (murny) [inline Gemfile]: Like this:

https://github.com/ualbertalib/HydraNorth/blob/496dcc631f7efc4b028b6f3515b45207182ab90b/Gemfile#L21-L22

**Comment by mbarnett** [comment]
@pbinkley we should cut a release of hydranorth when we have a chance to resolve this gemnasium alert

**Comment by pbinkley** [comment]
OK - we can do that Monday, when Henry's back to deploy to test

**Pull request merged**